h2/src/main/org/h2/mvstore/tx/Transaction.java

```diff
@@ -853,4 +862,5 @@ private static long composeState(int status, long logId, boolean hasRollback) {
         }
         return ((long)status << LOG_ID_BITS1) | logId;
     }
+
```

Comment:
we are not paid by number of sorce code lines, are we? :smile_cat:

Reply:
The new method was created at that position by IDE, but I moved it closer to other public methods and didn't notice the leftover line.